index.js

```diff
@@ -69,6 +78,16 @@ Shopify.prototype.request = function request(url, method, key, params) {
   return got(options).then(res => {
     const body = res.body;
 
+    const callLimitHeader = res.headers['x-shopify-shop-api-call-limit'];
+
+    if (typeof callLimitHeader === 'string') {
```

Comment:
This can only be `undefined` or a string right? In that case I would go with just `callLimitHeader`.